goal_src/engine/camera/cam-update.gc

```diff
@@ -5,3 +5,735 @@
+               )
+              (let ((a1-8 (-> s4-0 length)))
+               (set! arg3 (-> s4-0 from-bsp unk-data-2 0))
+               (if (< a1-8 arg3)
+                (set! v1-32 #f)
+                )
+               )
+              )
+             (when s3-0
+              (set!
+               a0-16
+               (nonzero? (logand (shl #x8000 16) (the-as int (-> s3-0 flags))))
+               )
+              (let ((a1-15 (-> s3-0 length)))
+               (set! arg3 (-> s3-0 from-bsp unk-data-2 0))
+               (if (< a1-15 arg3)
+                (set! a0-16 #f)
+                )
+               )
+              )
+             v1-32
+             )
+            )
+           (if (!= (-> s5-1 all-visible?) 'loading)
+            (set! (-> s5-1 all-visible?) #f)
+            )
+           (let* ((a0-19 s5-1)
+                  (t9-2 (method-of-object a0-19 dummy-16))
+                  )
+            (-> s4-0 ramdisk)
+            (set! arg3 (-> s4-0 string-block))
+            (when (t9-2 a0-19)
+             (countdown (v1-40 8)
+              (let ((a0-22 (-> s5-1 vis-info v1-40)))
+               (when a0-22
+                (if (!= a0-22 s4-0)
+                 (set! (-> a0-22 current-vis-string) (the-as uint -1))
+                 )
+                )
+               )
+              )
+             (set! (-> s5-1 all-visible?) #f)
+             )
+            )
+           )
+          (else
+           (cond
+            (a0-16
+             (if (!= (-> s5-1 all-visible?) 'loading)
+              (set! (-> s5-1 all-visible?) #f)
+              )
+             (let* ((a0-24 s5-1)
+                    (t9-3 (method-of-object a0-24 dummy-16))
+                    )
+              (-> s3-0 ramdisk)
+              (set! arg3 (-> s3-0 string-block))
+              (when (t9-3 a0-24)
+               (countdown (v1-50 8)
+                (let ((a0-27 (-> s5-1 vis-info v1-50)))
+                 (when a0-27
+                  (if (!= a0-27 s3-0)
+                   (set! (-> a0-27 current-vis-string) (the-as uint -1))
+                   )
+                  )
+                 )
+                )
+               (set! (-> s5-1 all-visible?) #f)
+               )
+              )
+             )
+            (else
+             (cond
+              ((and (= (-> s5-1 all-visible?) 'loading) (-> *level* play?))
+               )
+              (else
+               (quad-copy!
+                (-> s5-1 vis-bits)
+                (-> s5-1 bsp all-visible-list)
+                a2-1
+                )
+               )
+              )
+             )
+            )
+           )
+          )
+         )
+        (else
+         (quad-copy! (-> s5-1 vis-bits) (-> s5-1 bsp all-visible-list) a2-1)
+         )
+        )
+       )
+      (if (and *stats-bsp* (= *master-mode* 'game))
+       (format *stdcon* "~0kleaf-index = ~d~%" (-> s5-1 bsp unk-data-2 0) arg3)
```

Comment:
This probably had an error/warning with 1 arg because this format string is incorrectly recognized as taking 2 arguments.